### ISSUE TITLE
EventTimestampQualificationEnum duplicates information specified in the business event

### ISSUE THREAD

**Issue by hugohills-regnosys** [finos/common-domain-model#2073]
The `EventTimestampQualificationEnum` specified on `WorkflowStep -> timestamp -> qualification` duplicates event type that is specified in the business event, e.g., if the business event is a clearing event, then it is duplicative to specify `clearingConfirmationDateTime` rather than say just `Confirmation` timestamp.

Current enum:
```
enum EventTimestampQualificationEnum:
	clearingDateTime 
	clearingConfirmationDateTime
	clearingSubmissionDateTime 
	confirmationDateTime 
	eventCreationDateTime 
	eventExpirationDateTime 
	eventProcessingDateTime 
	eventSentDateTime 
	eventSubmittedDateTime 
	executionDateTime 
	transactionCreationDateTime
```
A revised enum would not reference the event, and could be something like:
```
enum TimestampQualificationEnum:
	Confirmation
	Creation
	Expiration
	Processing
	Sent
	Submission
```

**Comment by tomhealey-icma** [comment]
I agree with this approach. We just need some definition for each.

**Comment by tomhealey-icma** [comment]
I agree with this approach. We just need some definition for each.

**Comment by iansloyan** [comment]
I also agree with the proposed approach - but I have a question - I get that the eventTimeStamp is important and thus needs qualification but is there not a 1to1 relationship with another Enum e.g. EventIntentEnum or something like that? How does the qualification work?

**Comment by hugohills-regnosys** [comment]
> I also agree with the proposed approach - but I have a question - I get that the eventTimeStamp is important and thus needs qualification but is there not a 1to1 relationship with another Enum e.g. EventIntentEnum or something like that? How does the qualification work?

Yes - the `EventIntentEnum` is, in some (or maybe most) cases, used in the qualification of events.  

If the event type (e.g. clearing, execution) is removed from the `EventTimestampQualificationEnum`, the remaining part of the timestamp qualification (e.g. Creation, Expiration, Confirmation etc) can be combined with either the event qualification (or event intent enum) to represent the same information as before (but without the duplication).

I wonder if the enum needs renaming?  It is actually a `WorkflowStep` timestamp qualification, there will likely be multiple timestamp entries (or even multiple `WorkflowStep`) for the same business event.

**Comment by eteridvalishvili** [comment]
Hi @hugohills-regnosys Can we link this issue to the Contribution Review Working Group Board if this hasn't been resolved yet? Please confirm if it's okay to close.

**Comment by eteridvalishvili** [comment]
Hi @hugohills-regnosys Following up to see if we can close this issue.
